packages/visualizations/addon/templates/components/navi-visualizations/table.hbs

```diff
@@ -14,6 +14,17 @@
                 value=(readonly column.displayName)
                 focus-out=(action 'updateHeaderDisplayName' column)
               }}
+              {{#if (eq column.type 'metric') }}
```

Comment:
Do we want the dropdown enabled for `threshold` columns as well?

Reply:
@jkusa Could you please confirm this?